React/CoreModules/RCTAlertController.m

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <React/RCTUtils.h>
+
+#import "RCTAlertController.h"
+
+@interface RCTAlertController ()
+
+#if !TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+@property (nonatomic, strong) UIWindow *alertWindow;
+
+@end
+
+@implementation RCTAlertController
+
```

Comment:
…and a new `#if !TARGET_OS_OSX // [TODO(macOS ISS#2323203)` here.

Because without the `@implementation RCTAlertController` part we are making the promise that this class exists at runtime, but then not actually providing a runtime implementation.